zh.js

```diff
@@ -1,747 +1,747 @@
+windows.maps.header 自定义游戏
+windows.maps.none 没有找到游戏!
+windows.maps.name 游戏名称
+windows.maps.editor 游戏编辑器
+windows.maps.publish 发布游戏
+windows.maps.community 开发者社区
+windows.publish.map.header 发布或更新游戏
+windows.publish.map.thumbnail 没有缩略图
+windows.publish.map.paste 粘贴游戏数据
+windows.publish.map.update 发布/更新地图
+windows.publish.map.login 创建并上传您的地图！
+windows.theatre.header 影院模式
+windows.theatre.record 启用录制
+windows.theatre.key 录制按键
+windows.clans.header 战队
+windows.clans.members 战队成员 ({0})
+windows.clans.requests 申请加入战队
+windows.clans.requests.none 没有申请
+windows.clans.page [{0}] 战队页面
+windows.clans.create 创建工战队
+windows.clans.name 战队名称
+windows.clans.view 浏览战队
+windows.clans.join 加入战队
+windows.clans.login 创建并加入战队!
+windows.store.header 商城
+windows.store.spin {0} 转盘（开箱）
+windows.store.agree 通过此次购买，您将同意 {0} 条款.
+windows.store.purchase 购买KR
+windows.store.voucher 兑换代金券
+windows.store.voucherinfo 从指定主播和活动获取代金券
+windows.store.market 市场
+windows.store.market2 Krunker市场
+windows.store.premium 购买Premium
+windows.store.premium2 Krunker Premium
+windows.store.buyP 在购买Premium后，您将可以自定义您的游戏用户名+体验更多专用功能！
+windows.store.buy 买卖物品
+windows.store.wheels 转盘
+windows.store.unlock 解锁新武器和物品
+windows.store.amount 账号中还有｛0｝KR
+windows.store.login 用于在商店中购买和解锁物品!
+windows.store.purchasing 购买中...
+windows.store.unboxed {0}开箱获得了{1}
+windows.trade.header 交易
+windows.scope.header 选择您的狙击镜准心
+windows.premium.header Premium
+windows.skin.header 选择皮肤
+windows.hat.header 选择头部
+windows.back.header 选择背部饰品
+windows.waist.header 选择腰部饰品
+windows.face.header 选择表情
+windows.melee.header 选择近战武器
+windows.attach.header 选择附件
+windows.reticle.header 选择准星
+windows.reticle.unlocked 于{0}级解锁
+windows.reticle.req 需要{0}级
+windows.dye.header 选择衣服
+windows.dye.default 默认衣服
+windows.dye.spin 衣服
+windows.changelog.header 变更日志
+windows.mods.community.header 社区模组
+windows.mods.community.fav 收藏
+windows.mods.community.new 最新模组
+windows.mods.community.hot 最热门
+windows.mods.community.name 模组名称
+windows.mods.community.none 没有找到该模组
+windows.publish.mod.header 发表模组
+windows.publish.mod.thumbnail 没有缩略图
+windows.publish.mod.name 模组名称
+windows.publish.mod.paste 模组网址
+windows.publish.mod.update 发布/更新
+windows.publish.mod.login 请登录以创建并上传!
+windows.secondary.header 选择副武器
+windows.secondary.unlocked 已解锁
+windows.secondary.req 需要至少{0}级
+windows.advertise.header 广告
+windows.support.header 支持
+windows.players.header 玩家列表
+windows.players.none 当前没有玩家...
+windows.join.header 加入
+windows.join.code 输入房间网址
+windows.client.header 客户端
+windows.client.windows Windows游戏安装包
+windows.client.mac macOS游戏安装包
+windows.client.linux Linux游戏安装包
+windows.ranked.header 排位赛
+windows.ranked.login 登录以进行排位赛!
+windows.ranked.level 您至少需要 <span color="black"> {0} 级</span> 以进行排位赛! 当前您的等级是 <span color="black"> {1} 级</span>.
+windows.ranked.mode.r1v1 单排
+windows.ranked.mode.r2v2 双排
+windows.ranked.mode.r4v4 四排
+windows.chall.header 挑战
+windows.chall.login 登录以完成挑战!
+thumbnail.upload.limit 上传失败. 最大缩略图限制为｛0｝KB。
+image.upload.limit 上传失败，最大图片限制为｛0｝KB。

```

Comment:
Just noticed that you've used: `。` here, and other parts of the language file has `.`

Can you please update these?